tcf_website/templates/search/search.html

```diff
@@ -58,7 +58,7 @@ <h3 class="mr-3">Courses</h3>
                 {% for dept, dept_data in courses.items %}
                 <div class="row mb-5">
                   <div class="col-12">
-                    <h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ dept_data.subdept_name }}</a></h5>
+                    <!--<h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ dept_data.subdept_name }}</a></h5>
```

Comment:
Once this PR gets approved to merge you can just remove this commented out code; it's already being done by what you have written but I understand you're probably keeping it for now to be safe.